CHANGELOG.md

```diff
@@ -16,23 +16,25 @@ Given the `MAJOR.MINOR.PATCH` pattern, here is how we decide to increment:
 ## Unreleased
 
 ## Added
-
+- Ability to use Social Media molecule as a Wagtail module in the Sidefoot
```

Comment:
Moved this to the correct position. The merge logic didn't keep it in the Unreleased section after several releases happened while the PR containing this change waited to be merged.